atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -143,11 +143,15 @@ def set_help(self, extra, worker_metadatas):
                 extra['image']['help'] = None
 
     def set_media_types(self, extra, worker_metadatas):
-        for platform in worker_metadatas:
-            annotations = get_worker_build_info(self.workflow, platform).build.get_annotations()
-            if annotations.get('media-types'):
-                extra['image']['media_types'] = json.loads(annotations['media-types'])
-                return
+        pulp_pull_results = self.workflow.exit_results.get(PLUGIN_PULP_PULL_KEY)
+        if pulp_pull_results and pulp_pull_results[1]:
```

Comment:
If `pulp_pull` ran *at all* it means we're running arrangement version >= 4, where we cannot be certain there is only a single worker build. So I think this should be `if pulp_pull_results:`, and if no media types were reported then we set an empty list.